common/supplemental/likelySubtags.xml

```diff
@@ -3550,6 +3550,24 @@ not be patched by hand, as any changes made in that fashion may be lost.
 		<!--{ ?; Lanna; ? } => { Northern Thai; Lanna; Thailand }-->
 		<likelySubtag from="und_Laoo" to="lo_Laoo_LA"/>
 		<!--{ ?; Lao; ? } => { Lao; Lao; Laos }-->
+		<likelySubtag from="und_Laoo_AU" to="hnj_Laoo_AU"/>
```

Comment:
I think this is a bad result. Just from quick checks, it appears that there are far more Lao speakers in these countries than there are Hmong Njua speakers. It is just that we haven't had anyone submit data for those. I suggest that we remove these for now, and I'll file a ticket to work on a longer term solution.

Reply:
I opened a ticket for possible approach for the longer term.

Reply:
makes sense.

There are still some manual reverts here already. I can file a ticket with those to make sure they are tracked.

Reply:
removed, ptal